logspout/start.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env sh
+
+tag_csv=logspout
+if [[ "${audius_loggly_tags}" ]]; then
+   tag_csv=${tag_csv},${audius_loggly_tags}
+fi
+
+hostname=${audius_discprov_url}
```

Comment:
Could we just do an if else for if either of these hostname env vars are defined? This reads a little confusing